tests/unit/commands/test_remote.py

```diff
@@ -547,85 +551,84 @@ def test_architecture_in_project_metadata(
         pytest.param(
             "amd64,amd64,riscv64",
             ["amd64", "amd64", "riscv64"],
-            id="launchpad-handles-duplicates",
+            id="duplicates-passthrough-to-launchpad",
```

Comment:
Github's diff is unreadable.  For example, I didn't change this line. I deleted some tests and Github mixing up similar tests